eclass/distutils-r1.eclass

```diff
@@ -820,6 +820,10 @@ distutils-r1_python_test() {
 		die "${FUNCNAME} can be only used after calling distutils_enable_tests"
 	fi
 
+	if [[ -z ${EPYTHON} ]]; then
```

Comment:
There are some other python* eclass functions that have similar check, please sync it to use the same syntax.

Reply:
Yep, I've saw those, but I decided to mimic the `if not-set; then die` pattern that is used in the function. Which is also the pattern I prefer. But I can change it back to `[[ -n ${EPYTHON} ]] || die "EPYTHON unset, invalid call context"`.